resources/assets/ts/Utilities.ts

```diff
@@ -1,4 +1,6 @@
-enum Club {
+import * as $ from "jquery"
+
+export enum Club {
     None,
     BcClub = 2,
```

Comment:
Why bc with a number, but the rest without?
Also, I thing this might not be optimal to hardcode section - can we instead fetch club id from the DB somehow?
The point is - I'd like to generalize this bit too, using only section IDs everywhere and not mentioning section names anywhere in the code.